### ISSUE TITLE
"Is the Enemy a Boss?" is not functioning properly.

### ISSUE THREAD

**Issue by Scarfuncle** [PathOfBuildingCommunity/PathOfBuilding#1450]
I am running version 1.4.170.11.

The "Is the enemy a Boss" tick in for effective DPS is not working for me. It does not show a difference between boss damage and "no." My pastebin is https://pastebin.com/M2BciauX . My main skill is shield charge. I have tried resetting my tree, rebuilding the build from scratch, taking off items, skills, etc and it changes nothing. This issue started on Sept 19 or 20.

Thank you for looking into this

**Comment by Wires77** [comment]
Enemies have armor, and armor is less effective against bigger hits.  This, combined with a couple sources of "Overwhelm" you have reduces the enemy's physical damage reduction to 0, and it can't go any lower than that.

You can see this for yourself by removing Awakened Brutality, Pulverise, and Ruthless from your links, changing the calculation mode for shield charge in the Calcs tab to "Effective DPS", and clicking on the OH Eff. DPS Mod row in the Physical column to see the breakdown:

![image](https://user-images.githubusercontent.com/1209372/94069176-c38e1d00-fdb5-11ea-8a5e-4404a4e9f4b6.png)

TL;DR: You do enough damage to ignore physical damage reduction against bosses, so they take the same amount of damage as white mobs.

**Comment by Scarfuncle** [comment]
I already tried removing all my sources of overwhelm. Once I do this, my no boss dps is 788,460 and my shaper damage is 771, 984. There is a difference, but it should be a big difference.  https://pastebin.com/qAYb4kEh for what I'm seeing.

**Comment by Wires77** [comment]
It isn't a big difference because of how armor works.  Armor is significantly less effective the more damage it's trying to mitigate.  This is true for both players and monsters.  This graph illustrates the issue pretty well: https://pathofexile.gamepedia.com/File:Armour-percentdmg3.png

Sirus has about 30k armor, and you do about 50k damage per hit in this build, so Sirus would mitigate only about 3-9% of that damage, and if we see the physical damage reduction number PoB gives you from the pastebin you linked, it's 6%, falling right in that range.
